lib/go-parser/index.ts

```diff
@@ -76,17 +76,8 @@ async function findGoBinaries(
           (section) => section.name === ".note.go.buildid",
         );
 
-        const interp = binaryFile.body.sections.find(
-          (section) => section.name === ".interp",
-        );
-
         if (!goBuildInfo && !goBuildId) {
           return resolve(undefined);
-        } else if (interp) {
```

Comment:
cc: @dkontorovskyy

Reply:
You've added some binaries compiled using cgo and they are passing with this section you mean? 
I'm also not sure, added Daniel in case he knows of a specific case 🤷‍♀️

Reply:
Decided to keep this `if` for now to reduce changes in this already big fix